src/App.js

```diff
@@ -0,0 +1,20 @@
+import { Route, Switch } from 'react-router-dom';
+import './App.css';
+import Books from './components/Books';
+import Categories from './components/Categories';
+import Header from './components/Header';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Switch>
+        <Route path="/" component={Books} exact />
+        <Route path="/books" component={Books} />
```

Comment:
- [ ] OPTIONAL - Consider removing the `path='/books'` as it returns the same component as the home route.